src/main/java/duke/Ui.java

```diff
@@ -5,24 +5,41 @@
 
 import java.util.Scanner;
 
+/**
+ * Ui class to manage user input and Duke output
+ */
 public class Ui {
     private static final String space = "     ";
     private static final String divider = "    -------------------------------------------->>>>>\n";
     public Scanner sc;
 
+    /**
+     * Ui constructor
+     */
     public Ui() {
         this.sc = new Scanner(System.in);
     }
 
+    /**
```

Comment:
I found you code easy to understand and easy to read. The quality of code was good and you adhered to coding standards. The names of methods were appropriate as well. Good work and keep it up!